sky/clouds/kubernetes.py

```diff
@@ -162,6 +163,12 @@ class Kubernetes(clouds.Cloud):
                                                              'supported in '
                                                              'Kubernetes.',
     }
+    _CLOUD_EXCLUDABLE_FEATURES: Dict[
+        clouds.CloudImplementationFeatures, ExcludableFeatureCheckProtocol] = {
+            clouds.CloudImplementationFeatures.AUTOSTOP:
+                lambda config: config.cluster_name.startswith(
+                    'sky-spot-controller-')
```

Comment:
```suggestion
                lambda config: config.cluster_name == spot.SPOT_CONTROLLER_NAME
```

Reply:
I tried this but it was giving a circular import error

Reply:
Oh ic... Maybe you could copy-paste the `f'sky-spot-controller-{common_utils.get_user_hash()}'` to here? Or if @concretevitamin has any other ideas?

Reply:
Happy with either way. 

I just realized this PR is merging into the k8s branch. cc @romilbhardwaj to take a look too. On the surface it seems like we should make k8s.py importing spot work (the latter should not import k8s?).